### ISSUE TITLE
Overpass management

### ISSUE THREAD

**Issue by cchristelis** [hotosm/field-campaigner#78]
# Problem

Currently, we use the first user attempt to view a campaign as a signal to download data from overpass. Downloading data can take a lot of time causing a bad user experience. We should always serve cached data to the user and update the cache in the background.

# Proposed Solution

Set up a cron that cycles through projects and service the overpass requirements.

**Comment by cchristelis** [comment]
# UPDATED Proposed Solution

Rather than having a cron (adding another moving part to the system). We have the last data show, with a date and a note that it is being updated.

**Comment by smit1678** [comment]
@cchristelis Couple thoughts here: 

* Using only Overpass for monitoring will get us too much data and doesn't have all the changeset information we need to flag certain items - like type of editor used - to be able to filter out the data that we're tracking. 

* Should we look to slightly change the main data source? Instead of Overpass and working around it's limitations, we look to leverage the work from Mapbox around their OSMCHA tool. There is new work from them to expose a full API. 

* Look to run out own osmcha infrastructure so we can customize the types of features we want to flag. Instead of suspicious changesets, it's based on the insight functions that are part of the campaign.

**Comment by cchristelis** [comment]
@smit1678, let's use both of them in parallel.

**Comment by meomancer** [comment]
@smit1678 @cchristelis 
Osmcha feature already added as new insight function.
https://github.com/hotosm/field-campaigner/pull/104

**Comment by smit1678** [comment]
@meomancer @cchristelis Is the OSMCHA just an insight function? From our previous chats, OSMCHA was being seen as a data source and not just a insight function. If we look at it as a data source, we can rely less on Overpass for evaluating errors or evaluating completeness.

**Comment by meomancer** [comment]
hi @smit1678 
yes, i miss it, thank you
will update it

**Comment by meomancer** [comment]
new ticket for OSMCHA/Overpass management
https://github.com/hotosm/field-campaigner/issues/170
